examples/test_cpp/test_loop.cc

```diff
@@ -68,28 +67,54 @@ int test_loop::load_file(const std::string& file_name, std::string& config_str)
   return err::success;
 }
 
-std::string test_loop::generate_experiment_name(const std::string& experiment_name_base, size_t threads, size_t features, size_t actions, size_t index)
+std::string test_loop::generate_experiment_name(const std::string& experiment_name_base, size_t threads, size_t features, size_t actions, size_t slots, size_t index)
 {
-  return experiment_name_base + "-t" + std::to_string(threads) + "-f" + std::to_string(features) + "-a" + std::to_string(actions) + "-i" + std::to_string(index);
+  return experiment_name_base + "-t" + std::to_string(threads) + "-f" + std::to_string(features) + "-a" + std::to_string(actions) + "-s" + std::to_string(slots) + "-i" + std::to_string(index);
 }
 
-void test_loop::run() {
+void test_loop::run(bool is_ccb) {
```

Comment:
This whole tool is going to be weird once slates is in the mix

Reply:
slates are completely independent code path which is not going to invalidate neither cb nor ccb right?

Reply:
this is true, but I don't know if it is __completely__ different. This was more of a general comment, not something you need to address here